docs/section4/4-12.md

```diff
@@ -0,0 +1,43 @@
+# Airside System - Rule 4-12
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** True  
+**Rule ID:** 4-12  
+**Rule Description:** For Systems 6 and 8, only the terminal-unit fan and reheat coil shall be energized to meet heating set point during unoccupied hours.  
+**Rule Assertion:** B-RMR = Expected value   
+**Appendix G Section:** Section G3.1.2.4      
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Evaluate each HeatingVentilationAirConditioningSystem Data Group
+
+**Applicability Checks:** 
+
+1. Applicable to baseline HVAC systems 6 and 8 only. 
+
+**Function Calls:**  
+
+1. get_baseline_system_types()  
+2. get_hvac_zone_list_w_area()  
+
+## Rule Logic:  
+**Applicability Check 1:**  
+- Get dictionary for finding baseline system types for hvac systems: `baseline_hvac_sys_type_ids_dict_b = get_baseline_system_types(B_RMR)`
+- Get dictionary with list of zones associated with each HVAC system: `dict_hvac_sys_zones_served_b = get_hvac_zone_list_w_area(B_RMR)`
+- For each hvac system in the B_RMR: `for hvac_b in B_RMR...HeatingVentilationAirConditioningSystem:`
+    - Check if system type is of system type 6, 6b, 8, 8a, 8b, or 8c, if yes then carry on with check: `if baseline_hvac_sys_type_ids_dict_b.keys()[list(baseline_hvac_sys_type_ids_dict_b.values()).index(hvac_b.id)] in ["SYS-6, SYS-6b",SYS-8",SYS-8a",SYS-8b",SYS-8c"]:`
+        - Reset all terminals unoccupied equals cycling boolean variable to TRUE: `all_terminals_unocc_cycling = TRUE`
+        - Get hvac system fan system operation during unoccupied hours: `fan_system_operation_unocc = hvac_b.fan_system[0].operation_during_unoccupied`
+        - Get a list of zones served by the HVAC system: `zones_served_b = list(dict_hvac_sys_zones_served_b[hvac_b.id]["Zone_List"].values()))`
+        - Loop through each zone served by the hvac system: `For zone_b in zones_served_b:`
+            - Loop through each terminal unit associated with each zone and check the operation during unoccupied periods: `For terminal_b in zone_b.terminals:`
+                - Check if unoccupied operation is set to cycling, if not then set all_terminals_unocc_cycling boolean variable equal to FALSE: `if terminal_b.operation_during_unoccupied != "CYCLING": all_terminals_unocc_cycling = FALSE`     
+        
+        **Rule Assertion:**  
+        - Case 1: For applicable hvac system, if the hvac system fan_system_operation_unocc equals "Keep_off" and unoccupied fan control for each associated terminal unit equals "cycling" then pass: `if fan_system_operation_unocc == "KEEP_OFF" AND all_terminals_unocc_cycling == TRUE: outcome == "PASS"`
```

Comment:
Exception G3.1.2.4 applies only to meeting HEATING setpoint during unoccupied hours. For cooling setpoint, the fan_system_operation_unocc should be to cycle instead of KEEP_OFF. I don't think we currently have a way to capture difference in cooling vs heating unoccupied operation in the RMR.

Reply:
as discussed during the RDS call , we need additional inputs in the schema to characterize and we might not have relevant inputs to characterize the required staging. decided to make this non-mandatory